### ISSUE TITLE
score-voting

### ISSUE THREAD

**Issue by MellowVoidUser** [nextcloud/polls#765]
I would love to have a score-voting poll for many nuanced single-winner (small-)group-decisions.
see http://scorevoting.net/
It can be argued this already exists, but currently it's limited to yes, no or maybe while I would like 0-10 or even 0-99 as possible scores.

**Comment by dartcafe** [comment]
What about ordering your score from high to low or vice versa?

**Comment by MellowVoidUser** [comment]
@dartcafe that would be cool and convenient! (but don't confuse ordinal voting methods with score-voting ones , see the theory and arguments on http://scorevoting.net/ )

**Comment by aliceinwire** [comment]
I also need this feature for organizing Gentoo GSoC

**Comment by chris-blues** [comment]
I'm also looking for a poll in a score method. a 5-star ranking method would be enough in my case.

Would be pretty amazing if you people could enhance NC polls with more methods!

**Comment by JustACasual** [comment]
+ 1 for this!
Option to distribute a fixed number of votes over all available options would be great!

**Comment by dartcafe** [comment]
> Option to distribute a fixed number of votes over all available options would be great!

???
![grafik](https://github.com/nextcloud/polls/assets/26707476/aa87637e-25bb-4227-bc0d-4f2e3bc7e9d8)

**Comment by dartcafe** [comment]
Yes seems to be another type of voting. This would need a [separate issue](https://github.com/nextcloud/polls/issues/new?assignees=&labels=enhancement&projects=&template=FEATURE_REQUEST.yml)

**Comment by 4jNsY6fCVqZv** [comment]
An inspiration for the implementation of this new poll type from Loomio: https://help.loomio.com/en/user_manual/polls/proposal_types/index.html#score-poll
